src/metatensor/models/cli/conf/architecture/soap_bpnn.yaml

```diff
@@ -25,6 +25,7 @@ model:
     activation_function: SiLU
 
 training:
+  device: "cpu"
```

Comment:
This should be in the root of the yaml file and not here. We should parse this through I think or can you get this maybe from the dataset if we initialize them correctly?

Reply:
I think it should be here. Talk to me